lib/cache.rb

```diff
@@ -74,17 +74,16 @@ def delete(name)
 
   def fetch(name, expires_in: nil, force: nil, &blk)
     if !block_given?
-      if force
-        raise ArgumentError,
-              "Missing block: Calling `Cache#fetch` with `force: true` requires a block."
-      end
+      raise ArgumentError, "Calling `Cache#fetch` with `force: true` requires a block." if force
```

Comment:
Shortened the description of the error message to have prettier put it on its own line. Totally unrelated to the fix.